package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "color-info",
   "displayName": "Color Info",
-  "description": "Provides quick information about css colors",
+  "description": "Provides quick information about colors in any programming language",
```

Comment:
Can you also please revert this change this it is not strictly related to the PR